sdk/storage/Azure.Storage.DataMovement/tests/TransferManagerTests.cs

```diff
@@ -151,6 +154,13 @@ public async Task BasicItemTransfer(
             srcResource.VerifyNoOtherCalls();
             dstResource.VerifyNoOtherCalls();
         }
+        foreach (MemoryTransferCheckpointer.Job job in checkpointer.Jobs.Values)
+        {
+            Assert.That(job.Parts.Count, Is.EqualTo(1), "Items should be single-part.");
```

Comment:
A little nitpicky but I think you can also assert Job part's status here as well. I believe it should be in 'Queued' state at this point in time.